ElementX/Sources/Application/AppSettings.swift

```
@@ -223,10 +223,10 @@ final class AppSettings {
         
     // MARK: - Bug report
 
-    let bugReportServiceBaseURL: URL! = URL(string: Secrets.rageshakeServerURL)
+    let bugReportServiceBaseURL: URL? = URL(string: Secrets.rageshakeServerURL)! // swiftlint:disable:this force_unwrapping
```

What's the plan here, will the secrets eventually become optional themselves and we stop force unwrapping?

---

Correct, the secrets will become optional so we could change this to
```
let bugReportServiceBaseURL: URL? = Secrets.rageshakeServerURL.map { URL(string: $0)! }
```
I.e. fail early on a secret assuming it is a valid value at build time, otherwise do nothing.